config/win/bcc.mk

```diff
@@ -162,11 +181,63 @@ define create_dynlib
    $(if $(wildcard __dyn__.tmp),@$(RM) __dyn__.tmp,)
    $(foreach file,$^,$(dynlib_object))
    @$(ECHO) $(ECHOQUOTE), $(subst /,$(ECHOBACKSLASH),$(DYN_DIR)/$@),, $(subst /,$(ECHOBACKSLASH),$(DLIBS))$(ECHOQUOTE) >> __dyn__.tmp
-   $(DY) $(DFLAGS) $(HB_USER_DFLAGS) c0d32.obj @__dyn__.tmp
+   $(DY) $(DFLAGS) $(HB_USER_DFLAGS) $(DLL_STARTUP_MODULE) @__dyn__.tmp
    @$(CP) $(subst /,$(DIRSEP),$(DYN_DIR)/$(basename $@)$(LIB_EXT)) $(subst /,$(DIRSEP),$(IMP_FILE))
    @$(RM) $(subst /,$(DIRSEP),$(DYN_DIR)/$(basename $@)$(LIB_EXT))
 endef
 
 DY_RULE = $(create_dynlib)
 
+
+# Don't let Clang-based (7.60+ (?)) bcc64 parse UTF-8 in our sources.
+# https://quality.embarcadero.com/browse/RSP-26502 got closed but the issue remains:
+# if Clang-based bcc64 detects UTF-8, it will convert it to ANSI in the binary.
+# So, for known UTF-8 string literals in our sources, we modify CC_RULE:
+# first, a PowerShell script converts UTF-8 to \123 octal characters, and
+# then the result is fed into bcc64 instead of the original file.
+# PowerShell could be changed to C if anyone cares.
+# As the source location changes, -I$(GRANDP) must be added as well.
```

Comment:
It's hard to believe that a modern compiler needs a hack like that. "bcc64" is a commercial rebadge of llvm/clang, and the latter is capable of compiling UTF-8 strings with no issues by default.

According to this blog article, the relevant compiler option might be `--codepage:65001`:
https://blogs.embarcadero.com/the-delphi-compiler-and-utf-8-encoded-source-code-files-with-no-bom/

But why not use the native llvm/clang? (e.g. llvm-mingw, which has UCRT and ARM64 support out of the box, open-source, free and runs on all major platforms: https://github.com/mstorsjo/llvm-mingw/releases)